autoscheduler/scraper/management/commands/scrape_departments.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Department
+
+
+class Command(base.BaseCommand):
+    """ Gets all departments from banner and adds them to the database """
+
+    def add_arguments(self, parser):
+        parser.add_argument('--term', type=str, required=False, default="201931")
+
+    def handle(self, *args, **options):
+        pass
```

Comment:
While I do prefer the file to be called `scrape_depts` rather than `scrape_departments` since it's less to type, why make a completely different file instead of either using this one or renaming it? 

Either delete this file, or move your changes from `scrape_depts` into it and delete it